pod.go

```diff
@@ -330,6 +331,13 @@ type PodConfig struct {
 	// Annotations keys must be unique strings and must be name-spaced
 	// with e.g. reverse domain notation (org.clearlinux.key).
 	Annotations map[string]string
+
+	// Devices is a linux specific configuration, these are devices that
+	// must be available within the container.
+	// Appears as a direct child of "linux" object in the oci json spec.
+	// Note this is different from the devices under "resources".
+	//Devices []CompatOCISpec.Linux.LinuxDevice
```

Comment:
Is this comment line meant to be here? If so, it would be better to put the at the end of the "oci json spec" sentence I think.

Reply:
This comment line should have been deleted. I have made the change.